tests/Our.Umbraco.Ditto.Tests/DictionaryValueTests.cs

```diff
@@ -18,19 +19,37 @@ public class MyModel
             public string Foo { get; set; }
         }
 
-        [Test]
-        public void DictionaryValue_Returned()
+        [TestFixtureSetUp]
+        public void Init()
         {
-            // Create mock dictionary items
-            var mockDict = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+            if (!CultureDictionaryFactoryResolver.HasCurrent)
             {
-                { "hello", "world" },
-                { "foo", "bar" }
-            };
+                var mockDictionary = new Mock<ICultureDictionary>();
+                mockDictionary.SetupGet(p => p["hello"]).Returns("world");
+                mockDictionary.SetupGet(p => p["Foo"]).Returns("bar");
+
+                var mockDictionaryFactory = new Mock<ICultureDictionaryFactory>();
+                mockDictionaryFactory.Setup(x => x.CreateDictionary()).Returns(mockDictionary.Object);
 
-            // Replace dictionary helper getter
-            UmbracoDictionaryHelper.GetValue = (key) => mockDict[key];
 
+                CultureDictionaryFactoryResolver.Current = new CultureDictionaryFactoryResolver(mockDictionaryFactory.Object);
+
+                Resolution.Freeze();
+            }
+        }
+        
+        [TestFixtureTearDown]
+        public void Teardown()
+        {
+            if (Resolution.IsFrozen)
+            {
+                Resolution.Reset();
```

Comment:
On a complete tangent, I had a problem with the frozen resolution on PR #209 ... and seeing this `Resolution.Reset()` might fix my issue... thanking you! 💯